{"name": "test_login_unsuccessful", "status": "failed", "statusDetails": {"message": "AssertionError", "trace": "self = <test_reqres_api_negative.Test_negative_verification object at 0x00000189F1B4B610>\n\n    @allure.description(\"Негативная проверка POST неудачный вход в систему\")\n    def test_login_unsuccessful(self):\n        print(\"Метод POST  неудачный вход в систему\")\n        result_post = Reqres_api.login_unsuccessful()\n>       Сhecking.check_status_code(result_post, 400)\\\n\ntests\\test_reqres_api_negative.py:92: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nresponse = <Response [201]>, status_code = 400\n\n    @staticmethod\n    def check_status_code(response: Response, status_code):\n>       assert status_code == response.status_code\nE       AssertionError\n\nutils\\cheking.py:13: AssertionError"}, "description": "Негативная проверка POST неудачный вход в систему", "steps": [{"name": "POST", "status": "passed", "start": 1685391414171, "stop": 1685391414560}], "attachments": [{"name": "stdout", "source": "efe8f9fc-1163-4b0b-abd8-a8819644a4a7-attachment.txt", "type": "text/plain"}], "start": 1685391414171, "stop": 1685391414561, "uuid": "f55c68c6-8887-45f7-baf3-582bfb0d5e72", "historyId": "3340f5328637e150a698ca883226e5df", "testCaseId": "185c8bb9630c506aa996c80a3ee81b98", "fullName": "tests.test_reqres_api_negative.Test_negative_verification#test_login_unsuccessful", "labels": [{"name": "epic", "value": "Негативная проверка "}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_reqres_api_negative"}, {"name": "subSuite", "value": "Test_negative_verification"}, {"name": "host", "value": "HOME-PC"}, {"name": "thread", "value": "1600-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_reqres_api_negative"}]}